packages/tools/src/account/unlock.ts

```diff
@@ -334,6 +347,41 @@ function getSudtCellDep(): CellDep {
   };
 }
 
+async function getRollupCell(
+  godwokenClient: Godwoken,
+  ckbRpc: RPC,
+  rollupTypeHash: Hash
+): Promise<Cell> {
+  const result = await godwokenClient.getLastSubmittedInfo();
+  const txHash = result.transaction_hash;
+  const tx = await ckbRpc.get_transaction(txHash);
+
+  if (tx == null) {
```

Comment:
We can wait & retry times since the tx may need few seconds to spread over the CKB network.

Reply:
Retry every second, a total of ten times now